README.md

```diff
@@ -173,6 +173,11 @@ $ make bootstrap
 ...
 ```
 
+Nomad creates many file handles for communicating with tasks, log handlers, etc.
+In some development environments, particularly macOS, the default number of file
+descriptors is too small to run Nomad's test suite. You should set
+`ulimit -n 1024` or higher in your shell.
```

Comment:
Might be worth specifying that this is scoped to the shell it runs on and doesn't affect other running or future shells.